nix/.stack.nix/cardano-sl-db.nix

```diff
@@ -44,6 +44,7 @@
           (hsPkgs.ekg-core)
           (hsPkgs.ether)
           (hsPkgs.exceptions)
+          (hsPkgs.extra)
```

Comment:
Not sure about this one. This was introduced by @angerman few weeks ago directly onto develop. It's a new file that doesn't even exist on release/2.0.0 (cf: https://github.com/input-output-hk/cardano-sl/tree/release/2.0.0/nix/.stack.nix/cardano-sl-db.nix)!